src/main/python/gemini/cosimulation/nrel_spmc_controller/rudimentary_spmc_v5.py

```diff
@@ -0,0 +1,85 @@
+import math
+import numpy as np
+
+#=========================================================
+#      SPM control algorithm starts here
+#      objective: minimize peak power
+#=========================================================     
+class SPM_Control():
+    def __init__(self, time_step_mins, max_power_evse, min_power_evse):
+        self.time_step_mins = 1                   # time step: 1 minutes
```

Comment:
This is ignoring the parameter time_step_mins. Is this intentional?

Reply:
fixed